addons/website_sale/views/templates.xml

```diff
@@ -1921,14 +1921,17 @@
             <p class="text-muted">
                 <t t-foreach='website.shop_extra_field_ids' t-as='field' t-if='product[field.name]'>
                     <b><t t-esc='field.label'/>: </b>
-                    <t t-if='field.field_id.ttype != "binary"'>
-                        <span t-esc='product[field.name]' t-options="{'widget': field.field_id.ttype}"/>
+                    <t t-if='field.field_id.ttype == "selection"'>
+                        <span t-esc='product[field.name]' t-options="{'widget': 'text'}"/>
```

Comment:
` <span t-esc='product[field.name]' t-options="{'widget': 'selection', 'selection': dict(product._fields[field.name].selection)}"/>` should works ;)

Reply:
maybe that we should only support char field and binary :/ It was the initial idea...
We can have bug on each type of field for now... :/
I can try to handle other case internally if you want.

Reply:
> maybe that we should only support char field and binary :/ It was the initial idea...
> We can have bug on each type of field for now... :/

So, if I understand correctly, it would becomes:
`
<t t-if='field.field_id.ttype == "binary"'>
<t t-elif='field.field_id.ttype == "char"'>
`

> I can try to handle other case internally if you want.

It would be nice. Thanks!

Reply:
No, I means on website, where you select fields ;) we could just allow char and binary field...

I dont know what prefer @nim-odoo 
add feature for all type of field, or just 'dicrease' the power of the feature and restrict types allowed. (Seems used when I see the ticket ;) )  since we are in stable.

In v14 poeple who want selection field should for example create new computed char field to use it.

Reply:
> No, I means on website, where you select fields ;) we could just allow char and binary field...

Indeed. :)
Then customer will probably need to remove some unsupported fields from website configuration.
But at least, no change will be required in the view.

Reply:
let's restrict to the types allowed :)